index.js

```diff
@@ -176,6 +188,22 @@ class LiveStrategyExecution extends EventEmitter {
     this.processing = false
   }
 
+  _emitStrategyExecutionResults (type = 'candle', data) {
+    const { symbol } = this.strategyOpts
+    const { candlePrice } = this.strategyState
+    const price = type === 'candle' ? data[candlePrice] : data.price
+
+    let hasOpenPosition = false
```

Comment:
this extra variable is not required as openPosition is already nullable and the if checks that